src/synopsys-action/tools-parameter.ts

```diff
@@ -99,6 +99,42 @@ export class SynopsysToolsParameter {
       polData.data.polaris.prComment = {enabled: false}
     }
 
+    if (parseToBoolean(inputs.POLARIS_REPORTS_SARIF_CREATE)) {
+      const sarifReportFilterSeverities: string[] = []
+      const sarifReportFilterAssessmentIssuesType: string[] = []
+
+      if (inputs.POLARIS_REPORTS_SARIF_SEVERITIES != null && inputs.POLARIS_REPORTS_SARIF_SEVERITIES.length > 0) {
```

Comment:
we can directly check if (inputs.POLARIS_REPORTS_SARIF_SEVERITIES). Do we really need this?

Reply:
I have updated the code. Thanks.